pkg/policyreport/reportcontroller.go

```diff
@@ -208,8 +208,12 @@ func (g *ReportGenerator) updateClusterReportChangeRequest(old interface{}, cur
 }
 
 func (g *ReportGenerator) deletePolicyReport(obj interface{}) {
-	report := obj.(*report.PolicyReport)
-	g.log.V(2).Info("PolicyReport deleted", "name", report.GetName())
+	report, ok := kubeutils.GetObjectWithTombstone(obj).(*report.PolicyReport)
+	if ok {
+		g.log.V(2).Info("PolicyReport deleted", "name", report.GetName())
+	} else {
+		g.log.Info("Failed to get deleted object", "obj", obj)
```

Comment:
Is it ok to continue on failure?

Reply:
Probably yes, we don't care about the deleted object here.

Reply:
Right, should we return instead?

Reply:
I don't think so, we know something changed, we should notify the change.

Reply:
I checked the code, it triggers a reconciliation of policy reports. It should be fine to continue on error.